scripts/linters/pre_commit_linter.py

```diff
@@ -222,16 +223,17 @@ def _lint_all_files(
     custom_linters = []
     third_party_linters = []
 
-    js_ts_file_extension = '.js' in file_extension_type or (
-        '.ts' in file_extension_type)
-    py_file_extension = '.py' in file_extension_type
-    html_file_extension = '.html' in file_extension_type
-    css_file_extension = '.css' in file_extension_type
+    js_ts_file_extension = '*.js' in file_extension_type or (
```

Comment:
Something wrong with the naming here -- should be file_extension_types (plural). Or maybe better, file_extensions_to_lint.

Reply:
Done.